### ISSUE TITLE
Gosec not available yet?

### ISSUE THREAD

**Issue by nickbabkin** [ShiftLeftSecurity/sast-scan#76]
Hi, and thanks for the wonderful tool!

I wonder if gosec and staticcheck are already available?
I'm triggering a build in CircleCI with the following config:

`# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
# Comment
version: 2.1
# Use a package of configuration called an orb.
#orbs:
  # Security scan orb
  #sast-scan: appthreat/sast-scan@1.3.2
# Orchestrate or schedule a set of jobs
jobs:
  build:
    docker:
      - image: shiftleft/sast-scan
    working_directory: ~/repo
    steps:
      - checkout
      - run:
          name: Perform ShiftLeft Scan
          command: |
            scan -t gosec,staticcheck
      - store_artifacts:
          path: reports
          destination: sast-scan-reports
`

But build fails with the following output:
`INFO [2020-06-03 16:07:31,513] Scanning /root/repo using plugins ['gosec', 'staticcheck']
ERROR [2020-06-03 16:07:31,513] module '__main__' has no attribute 'gosec_scan'
WARNING [2020-06-03 16:07:31,513] Scan using the gosec plugin did not produce valid result
ERROR [2020-06-03 16:07:31,513] module '__main__' has no attribute 'staticcheck_scan'
WARNING [2020-06-03 16:07:31,513] Scan using the staticcheck plugin did not produce valid result`

Thank you!

**Comment by prabhu** [comment]
@nickbabkin Thank you.

`-t` parameter in scan indicates the language type. If you specify `-t go` then internally gosec and staticcheck would be used.

I will update the docs to clarify this as well.

**Comment by prabhu** [comment]
Also since staticcheck requires the compiled binary either do a go build before scan or add the parameter `--build`

**Comment by nickbabkin** [comment]
@prabhu thanks for the quick answer!
I added `-t go ` and it seems to execute fine now.

P.S. Funny enough, neither gosec nor staticcheck find any critical or high vulnerabilities in the following project: 

https://github.com/0c34/govwa

Here are the generated reports:
https://send.firefox.com/download/9b6ff8ce165f4757/#qbgxCAVv2EmeP1tBrLhE1Q


Wonder if it's a tool issue or sast-scan issue.

**Comment by prabhu** [comment]
yes, staticcheck is complaining about missing dependencies. I tried doing a go get and found that the project is quite ancient - 2 years is like 2 decades in go.

Try this sample project - https://github.com/HooliCorp/shiftleft-go-example
